metalinter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Linters": {
+	      "errcheck": "errcheck -abspath -ignore 'Close|Write|Serve' {path}:PATH:LINE:COL:MESSAGE",
+	      "license": "grep -rL 'Apache License' {path}:^(?P<path>.*?\\.go.*)"
+             },
+  "Severity": {"license": "error"},
+  "Enable": ["gofmt", "golint", "vet", "ineffassign", "misspell", "gocyclo"],
+  "Vendor": true,
+  "VendoredLinters": true,
```

Comment:
I'm assuming `Vendor` means use code in vendor directory, right? Does `VendoredLinters` mean that the linters are in the vendor directory? Just out of curiosity.